package.json

```diff
@@ -16,8 +16,7 @@
     "ios"
   ],
   "dependencies": {
-    "react-native-vector-icons": ">=1.0.4",
```

Comment:
react-native-vector-icons is not touched and all the time is in dependency, only last character was removed from this line.
Please check again my PR and correct me if I'm wrong.